src/views/index.njk

```diff
@@ -0,0 +1,97 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
```

Comment:
I suggest we have one whitespace before import keyword. At this moment we have mixed formatting. Aligning all import statement will be difficult so to avoid manual tweaking I suggest we do single space only.